apps/dashboard/app/(app)/desktop-sidebar.tsx

```diff
@@ -119,7 +119,6 @@ export const DesktopSidebar: React.FC<Props> = ({ workspace, className }) => {
       href: "/semantic-cache",
       label: "Semantic Cache",
       active: segments.at(0) === "semantic-cache",
-      hidden: true,
```

Comment:
please hide again until we are ready, you could use `hidden: process.env.NODE_ENV === "production"`

Reply:
done